src/lib/Preview.js

```diff
@@ -1245,8 +1250,10 @@ class Preview extends EventEmitter {
      */
     finishLoading(data = {}) {
         if (this.file && this.file.id) {
-            const tag = Timer.createTag(this.file.id, LOAD_METRIC.fullDocumentLoadTime);
-            Timer.stop(tag);
+            const contentLoadTag = Timer.createTag(this.file.id, LOAD_METRIC.fullDocumentLoadTime);
```

Comment:
Should this move to finshLoading of docBase?

Reply:
We want to track load times across all file types, not just documents

Reply:
sounds good after rename